new-lamassu-admin/src/pages/Transactions/Transactions.js

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from 'react'
+import moment from 'moment'
+import BigNumber from 'bignumber.js'
+import { startCase, lowerCase } from 'lodash/fp'
+import { makeStyles } from '@material-ui/core/styles'
+import useAxios from '@use-hooks/axios'
+
+import { mainStyles } from './Transactions.styles'
+import DetailsRow from './DetailsCard'
+import toUnit from './tx'
+
+import Title from '../../components/Title'
+import ExpTable from '../../components/expandable-table/ExpTable'
+import LogsDowloaderPopover from '../../components/LogsDownloaderPopper'
+import { FeatureButton } from '../../components/buttons'
+import { ReactComponent as TxInIcon } from '../../styling/icons/direction/cash-in.svg'
+import { ReactComponent as TxOutIcon } from '../../styling/icons/direction/cash-out.svg'
+import { ReactComponent as Download } from '../../styling/icons/button/download/zodiac.svg'
+import { ReactComponent as DownloadInverseIcon } from '../../styling/icons/button/download/white.svg'
+
+const Transactions = () => {
+  const [anchorEl, setAnchorEl] = useState(null)
+
+  const useStyles = makeStyles(mainStyles)
+
+  const classes = useStyles()
+
+  const { response: txResponse } = useAxios({
+    url: 'http://localhost:8070/api/txs/',
+    method: 'GET',
+    trigger: []
+  })
+
+  const formatCustomerName = (customer) => {
+    const { firstName, lastName } = customer
+
+    return `${startCase(lowerCase(firstName.slice(0, 1)))}. ${startCase(lowerCase(lastName))}`
```

Comment:
Why not do this:

```startCase(lowerCase(`${firstName.slice(0, 1)}. ${lastName}`))```

Reply:
`startCase` removes the dot after the first name.

Reply:
https://github.com/lamassu/lamassu-server/pull/336/commits/7ecd04765289b4680c8ff903ecee0d0da44af8e6 removes formatting the last name altogether, to prevent misformatting names with special characters, such as [Double-barreled names](https://en.wikipedia.org/wiki/Double-barrelled_name).